src/sql/util.py

```diff
@@ -266,6 +266,8 @@ def get_user_configs(file_path):
     """
```

Comment:
You may want to edit this docstring to reflect your new return values

Reply:
Made changes [here](https://github.com/ploomber/jupysql/pull/935/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R261-R268)

Reply:
@bryannho don't forget to check back your reviews. if you're ok with the changes, you must resolve the conversation

Reply:
@edublancas Messaged you on Slack!